src/Command/TemplateChangesCommand.php

```diff
@@ -193,7 +201,7 @@ static function (string $file) use ($targetDir): string {
 
     private function computeThemeTemplateFilesChangedAndOverridden(array $versionChangedFiles, string $themeName, bool $legacyMode): void
     {
-        $targetDir = $this->rootPath . 'themes' . \DIRECTORY_SEPARATOR . $themeName . \DIRECTORY_SEPARATOR;
+        $targetDir = $this->rootPath . $themeName . \DIRECTORY_SEPARATOR;
```

Comment:
I think we should check here if $themeName end with a / or not bifore concatenating it. Pheraps we could also remove at all the directory separator?

Reply:
An elegant way to do it is with

rtrim($themeName, \DIRECTORY_SEPARATOR) . \DIRECTORY_SEPARATOR

Reply:
Yes, absolutely